src/v2/spec.yaml

```diff
@@ -496,6 +496,10 @@ components:
           type: string
           description: Channel ID of the channel where the cast is to be posted. e.g. neynar, farcaster, warpcast
           example: "neynar"
+        idem:
+          type: string
+          description: Idempotency key
```

Comment:
```suggestion
          description: An Idempotency key to prevent duplicate publishing of events. Recommended format: 16-character string generated by the developer at the time of publishing. Use the same idem key on retry attempts. 
```